book/tutorials/self-signing-results.md

```diff
@@ -0,0 +1,209 @@
+
+```shell
+gpg --version
+```
+
+If GPG is installed, you should see something like the following:
+
+```shell
+gpg (GnuPG) 2.2.4
+libgcrypt 1.8.1
+Copyright (C) 2017 Free Software Foundation, Inc.
+License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
+This is free software: you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+
+Home: /home/ubuntu/.gnupg
+Supported algorithms:
+Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
+Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
+        CAMELLIA128, CAMELLIA192, CAMELLIA256
+Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
+Compression: Uncompressed, ZIP, ZLIB, BZIP2
+```
+
+If GPG is not installed on your machine, you'll receive a `command not found` error; if this is the case, you will need to install GPG before you can proceed.
+You can find instructions on how to natively install GPG on their website, linked [here](https://www.gnupg.org/download/).
+
+```{note}
+We have intentionally chosen not to include GPG in our QIIME 2 environment files.
+While installing and using GPG directly on your own machine is safe and recommended, distributing GPG through our Conda environments would require us (as software maintainers) to rely on additional third-party wrappers and distribution channels.
+These added layers of abstraction could expose the QIIME 2 {term}`Distribution` process to potential supply-chain or man-in-the-middle attacks.
+To minimize that risk, we ask users to install GPG directly from trusted system package sources.
+```
+
+Once GPG is installed on your machine, you will need to create a new keypair for use in {term}`signature` creation.
+If you already have a keypair you would like to use for this purpose, you can disregard the instructions below and skip to the {term}`signature` creation section.
+
+You can create a new GPG keypair using the following command:
+
+```shell
+gpg --full-generate-key
+```
+
+You will then be prompted to make selections on the type of keypair to create.
+Since the selections provided in GPG are subject to change over time, we have chosen not to go through them in this tutorial but we encourage you to review GPG's keypair creation documentation, which can be found [here](https://www.gnupg.org/gph/en/manual.html#AEN26).
+
+Once you have created your keypair, you will need to copy the keypair's fingerprint for use in {term}`signature` creation.
+You can obtain the fingerprint of your keypair by using the following command:
+
+```shell
+gpg --fingerprint
+```
+
+The output of this command should look something like this:
+
+```shell
+pub   ed25519 2025-09-24 [SC] [expires: 2027-09-24]
+      7B2B 042B D7F2 7AE4 DF86  18D0 80D7 286B 1D44 DB32
+uid           [ultimate] Example User <example@mail.me>
+sub   cv25519 2025-09-24 [E] [expires: 2027-09-24]
```

Comment:
I feel like this is confusing to phrasing to me because I can see my Key-Pair with the first command. 

your below context makes me better understand why I would do this. 

I might introduce this command as "This command shows all of the keypairs on your GPG keyring." or. "To see all the keypairs on your keyring run:"